### ISSUE TITLE
apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated

### ISSUE THREAD

**Issue by kkaempf** [epinio/epinio#204]
Deploying ServiceCatalog warns about `apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition`:
```
🚢 Deploying ServiceCatalog...............................................W0309 14:53:10.077453  130688 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

**Comment by agracey** [comment]
With all of these depreciations, what's the minimum version of K8s we want to support?

Also, this will require a bump to svc-cat itself https://github.com/kubernetes-sigs/service-catalog/blob/master/charts/catalog/templates/crds/clusterserviceclass.yaml#L1

**Comment by agracey** [comment]
https://github.com/kubernetes-sigs/service-catalog/issues/2809

**Comment by kkaempf** [comment]
This is fixed in latest Tekton (dashboard 0.15.0, pipeline 0.23.0)

**Comment by kkaempf** [comment]
> This is fixed in latest Tekton (dashboard 0.15.0, pipeline 0.23.0)

No, it's not :-(

**Comment by kkaempf** [comment]
It doesn't matter anymore since Epinio has an API.
